src/app/components/header/userMenu/userMenu.component.html

```diff
@@ -1,10 +1,16 @@
 <mat-menu #userMenu="matMenu">
+  <ng-template ngFor let-link [ngForOf]="links">
+    <a *ngIf="!link.guard || link.guard()" mat-menu-item [routerLink]="link.url" routerLinkActive="active">
+      <mat-icon *ngIf="link.icon">{{link.icon}}</mat-icon>
+      {{link.text}}
+    </a>
+  </ng-template>
   <button mat-menu-item (click)="logout()">
```

Comment:
Why do you use `a` elements for iterated links and `button` for logout button? Could you use the same in both elements?

Reply:
HTML elements have semantics, a is for links, buttons for actions that do not interact with the location